src/types/turbo.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * TypeScript type definitions for Turbo SDK web upload
+ * These types provide type safety for the Turbo upload operations in web environments
+ *
+ * Note: The Turbo SDK doesn't export these types for the web-specific uploadFile method,
+ * so we define them here based on the SDK's expected interface.
+ */
+import type { TurboUploadDataItemResponse } from '@ardrive/turbo-sdk/web';
+
+/**
+ * Progress information for file upload
+ */
+export interface TurboUploadProgress {
+  /** Total size of the file in bytes */
+  totalBytes: number;
+  /** Number of bytes processed so far */
+  processedBytes: number;
+}
+
+/**
+ * Event handlers for Turbo upload operations (web-specific)
+ */
+export interface TurboUploadEvents {
```

Comment:
These should probably be pulled from the sdk

Reply:
Fixed